butane_core/src/lib.rs

```diff
@@ -29,31 +28,28 @@ pub use sqlval::*;
 
 pub type Result<T> = std::result::Result<T, crate::Error>;
 
-/// Used internally by butane to track state about the object.
-///
-/// Includes information such as whether it has actually been created
-/// in the database yet. Butane automatically creates the field
-/// `state: ObjectState` on `#[model]` structs. When initializing the
-/// state field, use `ObjectState::default()`.
-#[derive(Clone, Debug, Default, Ord, PartialOrd)]
-pub struct ObjectState {
-    pub saved: bool,
+pub trait ToPkSql<T>
```

Comment:
docs missing

Reply:
Thanks for calling that out. My memory is going, and I can't seem to remember why I created this extra trait. Things seem to work without it, so I'm going to remove it...